hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.0__create_tables.sql

```diff
@@ -116,6 +116,7 @@ comment on table contract is 'Contract entity';
 create table if not exists contract_action
 (
     call_depth          integer                        not null,
+    call_operation_type smallint                       null,
```

Comment:
make it int to match the type from protobuf
```suggestion
    call_operation_type integer                       null,
```